### PR TITLE
Task to install rubygems to local system

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
# Description:

This PR adds a `rake install` task that installs the copy of rubygems to the users system. This is better than the `ruby setup.rb` alternative, because it's more similar to what actually happens in real life when users install `rubygems`.

# Tasks:

- [x] Describe the problem / feature
- [ ] Write tests
- [x] Write code to solve the problem
- [ ] Get code review from coworkers / friends

I will abide by the [code of conduct](https://github.com/rubygems/rubygems/blob/master/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -93,6 +93,11 @@ task :check_deprecations do
   end
 end
 
+desc "Install rubygems to local system"
+task :install => :package do
+  sh "gem install pkg/rubygems-update-#{v}.gem && update_rubygems"
```
Comment (bronzdoc) [inline Rakefile]: Should we use our own executable instead of the system executable?

Comment (deivid-rodriguez) [reply Rakefile]: @bronzdoc I did some work in https://github.com/rubygems/rubygems/compare/move_gem_to_exe to make it easier to always use the local version of the gem CLI for development tasks, but for now I would like to keep it consistent with the rest of the `Rakefile`, where system rubygems is used for all tasks.

Comment (bronzdoc) [reply Rakefile]: Fair 👍

**Comment by bronzdoc** [comment]
@bundlerbot r+

**Pull request merged**